### ISSUE TITLE
训练det报WARNING

### ISSUE THREAD

**Issue by xuxiansheng2018** [PaddlePaddle/PaddleOCR#10427]
配置文件：ch_PP-OCRv3_det_student.yml
预训练模型：ch_PP-OCRv3_det_distill_train/

Global:
  debug: false
  use_gpu: true
  epoch_num: 500
  log_smooth_window: 20
  print_batch_step: 10
  save_model_dir: ./output/rec_ppocr_v3_distillation
  save_epoch_step: 20
  eval_batch_step: [0, 100]
  cal_metric_during_train: true
  pretrained_model:    ./param/ch_PP-OCRv3_rec_train/best_accuracy  
  checkpoints: 
  save_inference_dir:
  use_visualdl: false
  infer_img: doc/imgs_words/ch/word_1.jpg
  character_dict_path: ppocr/utils/ppocr_keys_v1.txt
  max_text_length: &max_text_length 25
  infer_mode: false
  use_space_char: true
  distributed: true
  save_res_path: ./output/rec/predicts_ppocrv3_distillation.txt

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Piecewise
    decay_epochs : [700, 800]
    values : [0.0001, 0.00005]#0.0005, 0.00005  values : [0.0001, 0.00002]
    warmup_epoch: 5
  regularizer:
    name: L2
    factor: 0 #3.0e-05

Architecture:
  model_type: &model_type "rec"
  name: DistillationModel
  algorithm: Distillation
  Models:
    Teacher:
      pretrained:
      freeze_params: false
      return_all_feats: true
      model_type: *model_type
      algorithm: SVTR
      Transform:
      Backbone:
        name: MobileNetV1Enhance
        scale: 0.5
        last_conv_stride: [1, 2]
        last_pool_type: avg
      Head:
        name: MultiHead
        head_list:
          - CTCHead:
              Neck:
                name: svtr
                dims: 64
                depth: 2
                hidden_dims: 120
                use_guide: True
              Head:
                fc_decay: 0.00001
          - SARHead:
              enc_dim: 512
              max_text_length: *max_text_length
    Student:
      pretrained:
      freeze_params: false
      return_all_feats: true
      model_type: *model_type
      algorithm: SVTR
      Transform:
      Backbone:
        name: MobileNetV1Enhance
        scale: 0.5
        last_conv_stride: [1, 2]
        last_pool_type: avg
      Head:
        name: MultiHead
        head_list:
          - CTCHead:
              Neck:
                name: svtr
                dims: 64
                depth: 2
                hidden_dims: 120
                use_guide: True
              Head:
                fc_decay: 0.00001
          - SARHead:
              enc_dim: 512
              max_text_length: *max_text_length
Loss:
  name: CombinedLoss
  loss_config_list:
  - DistillationDMLLoss:
      weight: 1.0
      act: "softmax"
      use_log: true
      model_name_pairs:
      - ["Student", "Teacher"]
      key: head_out
      multi_head: True
      dis_head: ctc
      name: dml_ctc
  - DistillationDMLLoss:
      weight: 0.5
      act: "softmax"
      use_log: true
      model_name_pairs:
      - ["Student", "Teacher"]
      key: head_out
      multi_head: True
      dis_head: sar
      name: dml_sar
  - DistillationDistanceLoss:
      weight: 1.0
      mode: "l2"
      model_name_pairs:
      - ["Student", "Teacher"]
      key: backbone_out
  - DistillationCTCLoss:
      weight: 1.0
      model_name_list: ["Student", "Teacher"]
      key: head_out
      multi_head: True
  - DistillationSARLoss:
      weight: 1.0
      model_name_list: ["Student", "Teacher"]
      key: head_out
      multi_head: True

PostProcess:
  name: DistillationCTCLabelDecode
  model_name: ["Student", "Teacher"]
  key: head_out
  multi_head: True

Metric:
  name: DistillationMetric
  base_metric_name: RecMetric
  main_indicator: acc
  key: "Student"
  ignore_space: False

Train:
  dataset:
    name: SimpleDataSet
    data_dir: ./data/new_rec_data/train_rec/
    ext_op_transform_idx: 1
    label_file_list:
    - ./data/new_rec_data/train_rec/train.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - RecConAug:
        prob: 0.5
        ext_data_num: 2
        image_shape: [48, 320, 3]
    - RecAug:
    - MultiLabelEncode:
    - RecResizeImg:
        image_shape: [3, 48, 320]
    - KeepKeys:
        keep_keys:
        - image
        - label_ctc
        - label_sar
        - length
        - valid_ratio
  loader:
    shuffle: true
    batch_size_per_card: 32
    drop_last: true
    num_workers: 0
Eval:
  dataset:
    name: SimpleDataSet
    data_dir: ./data/new_rec_data/val_rec/
    label_file_list:
    - ./data/new_rec_data/val_rec/val.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - MultiLabelEncode:
    - RecResizeImg:
        image_shape: [3, 48, 320]
    - KeepKeys:
        keep_keys:
        - image
        - label_ctc
        - label_sar
        - length
        - valid_ratio
  loader:
    shuffle: false
    drop_last: false
    batch_size_per_card: 32
    num_workers: 0

**Comment by xuxiansheng2018** [comment]
警告如下，但是训练能进行
[2023/07/18 15:32:35] ppocr WARNING: The pretrained params Teacher.neck.pan_lat_conv.3.weight not in model
[2023/07/18 15:32:35] ppocr WARNING: The pretrained params Teacher.head.binarize.conv1.weight not in model
[2023/07/18 15:32:35] ppocr WARNING: The pretrained params Teacher.head.binarize.conv_bn1.weight not in model
[2023/07/18 15:32:35] ppocr WARNING: The pretrained params Teacher.head.binarize.conv_bn1.bias not in model
[2023/07/18 15:32:35] ppocr WARNING: The pretrained params Teacher.head.binarize.conv_bn1._mean not in model
[2023/07/18 15:32:35] ppocr WARNING: The pretrained params Teacher.head.binarize.conv_bn1._variance not in model
[2023/07/18 15:32:35] ppocr WARNING: The pretrained params Teacher.head.binarize.conv2.weight not in model
[2023/07/18 15:32:35] ppocr WARNING: The pretrained params Teacher.head.binarize.conv2.bias not in model
[2023/07/18 15:32:35] ppocr WARNING: The pretrained params Teacher.head.binarize.conv_bn2.weight not in model
[2023/07/18 15:32:35] ppocr WARNING: The pretrained params Teacher.head.binarize.conv_bn2.bias not in model
[2023/07/18 15:32:35] ppocr WARNING: The pretrained params Teacher.head.binarize.conv_bn2._mean not in model
[2023/07/18 15:32:35] ppocr WARNING: The pretrained params Teacher.head.binarize.conv_bn2._variance not in model
[2023/07/18 15:32:35] ppocr WARNING: The pretrained params Teacher.head.binarize.conv3.weight not in model
[2023/07/18 15:32:35] ppocr WARNING: The pretrained params Teacher.head.binarize.conv3.bias not in model
[2023/07/18 15:32:35] ppocr WARNING: The pretrained params Teacher.head.thresh.conv1.weight not in model
[2023/07/18 15:32:35] ppocr WARNING: The pretrained params Teacher.head.thresh.conv_bn1.weight not in model
[2023/07/18 15:32:35] ppocr WARNING: The pretrained params Teacher.head.thresh.conv_bn1.bias not in model

**Comment by UserWangZz** [comment]
该issue长时间未更新，暂将此issue关闭，如有需要可重新开启。

**Comment by lsch0lz** [comment]
I still have this problem when trying to fine-tune PP-OCRv4.
Config:
```
Architecture:
  name: DistillationModel
  algorithm: Distillation
  model_type: det
  Models:
    Student:
      model_type: det
      algorithm: DB
      Transform: null
      Backbone:
        name: PPLCNetV3
        scale: 0.75
        pretrained: false
        det: True
      Neck:
        name: RSEFPN
        out_channels: 96
        shortcut: true
      Head:
        name: DBHead
        k: 50
    Student2:
      pretrained: null
      model_type: det
      algorithm: DB
      Transform: null
      Backbone:
        name: PPLCNetV3
        scale: 0.75
        pretrained: true
        det: True
      Neck:
        name: RSEFPN
        out_channels: 96
        shortcut: true
      Head:
        name: DBHead
        k: 50
    Teacher:
      pretrained: https://paddleocr.bj.bcebos.com/PP-OCRv4/chinese/ch_PP-OCRv4_det_cml_teacher_pretrained/teacher.pdparams
      freeze_params: true
      return_all_feats: false
      model_type: det
      algorithm: DB
      Backbone:
        name: ResNet_vd
        in_channels: 3
        layers: 50
      Neck:
        name: LKPAN
        out_channels: 256
      Head:
        name: DBHead
        kernel_list:
        - 7
        - 2
        - 2
        k: 50
Loss:
  name: CombinedLoss
  loss_config_list:
  - DistillationDilaDBLoss:
      weight: 1.0
      model_name_pairs:
      - - Student
        - Teacher
      - - Student2
        - Teacher
      key: maps
      balance_loss: true
      main_loss_type: DiceLoss
      alpha: 5
      beta: 10
      ohem_ratio: 3
  - DistillationDMLLoss:
      model_name_pairs:
      - Student
      - Student2
      maps_name: thrink_maps
      weight: 1.0
      key: maps
  - DistillationDBLoss:
      weight: 1.0
      model_name_list:
      - Student
      - Student2
      balance_loss: true
      main_loss_type: DiceLoss
      alpha: 5
      beta: 10
      ohem_ratio: 3
Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.001
    warmup_epoch: 2
  regularizer:
    name: L2
    factor: 5.0e-05
PostProcess:
  name: DistillationDBPostProcess
  model_name:
  - Student
  key: head_out
  thresh: 0.3
  box_thresh: 0.6
  max_candidates: 1000
  unclip_ratio: 1.5
Metric:
  name: DistillationMetric
  base_metric_name: DetMetric
  main_indicator: hmean
  key: Student
Train:
  dataset:
    name: SimpleDataSet
    data_dir: /home/tmp/scholuka/PaddleOCR/data/ch4_training_images/
    label_file_list:
      - /home/tmp/scholuka/PaddleOCR/data/train_icdar2015_label.txt
    ratio_list: [1.0]
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - DetLabelEncode: null
    - IaaAugment:
        augmenter_args:
        - type: Fliplr
          args:
            p: 0.5
        - type: Affine
          args:
            rotate:
            - -10
            - 10
        - type: Resize
          args:
            size:
            - 0.5
            - 3
    - EastRandomCropData:
        size:
        - 640
        - 640
        max_tries: 50
        keep_ratio: true
    - MakeBorderMap:
        shrink_ratio: 0.4
        thresh_min: 0.3
        thresh_max: 0.7
        total_epoch: 500
    - MakeShrinkMap:
        shrink_ratio: 0.4
        min_text_size: 8
        total_epoch: 500
      - NormalizeImage:
          scale: 1./255.
          mean:
          - 0.485
          - 0.456
          - 0.406
          std:
          - 0.229
          - 0.224
          - 0.225
          order: hwc
      - ToCHWImage: null
      - KeepKeys:
          keep_keys:
          - image
          - threshold_map
          - threshold_mask
          - shrink_map
          - shrink_mask
    loader:
      shuffle: true
      drop_last: false
      batch_size_per_card: 16
      num_workers: 8
  Eval:
    dataset:
      name: SimpleDataSet
      data_dir: /home/tmp/scholuka/PaddleOCR/data/ch4_test_images/
      label_file_list:
3>      - /home/tmp/scholuka/PaddleOCR/data/test_icdar2015_label.txt
      transforms:
      - DecodeImage:
          img_mode: BGR
          channel_first: false
      - DetLabelEncode: null
      - DetResizeForTest:
          limit_side_len: 960
          limit_type: max
      - NormalizeImage:
          scale: 1./255.
(>        mean:
          - 0.485
          - 0.456
          - 0.406
          std:
          - 0.229
          - 0.224
          - 0.225
          order: hwc
      - ToCHWImage: null
      - KeepKeys:
          keep_keys:
          - image
          - shape
          - polys
          - ignore_tags
    loader:
      shuffle: false
      drop_last: false
      batch_size_per_card: 1
      num_workers: 2
profiler_options: null
```
Execution: ```python3 tools/train.py -c configs/det/ch_PP-OCRv4/ch_PP-OCRv4_det_cml.yml```

I used both the det_v4 trainer weights [LINK]([训练模型](https://paddleocr.bj.bcebos.com/PP-OCRv4/chinese/ch_PP-OCRv4_det_train.tar))
